src/sections/Listings/LIstings.tsx

```diff
@@ -40,10 +40,12 @@ export const Listings: React.FC<ListingsProps> = ({ title }) => {
     );
   }
   return (
-    <div className='p-4 dark:bg-neutral-500'>
-      <h2 className='mb-4 text-4xl text-text-color-1'>{title}</h2>
+    <div className='p-4'>
+      <h2 className='mb-4 text-4xl text-text-color-1 lg:tracking-widest dark:text-white'>
```

Comment:
can you apply default styles for sm and them `md` `lg` and `xl`

Reply:
https://github.com/igorizviekov/mt-booking_app/commit/d711e0e558074667d2f7f521670bfe51c17dca26